app/assets/javascripts/discourse/tests/acceptance/user-status-test.js

```diff
@@ -217,11 +217,18 @@ acceptance("User Status", function (needs) {
     await visit("/");
     await openUserStatusModal();
 
-    await fillIn(".user-status-description", userStatus);
+    await fillIn(".user-status-description", "off to <img src=''> dentist");
     await pickEmoji(userStatusEmoji);
     await click("#tap_tile_one_hour");
     await click(".btn-primary"); // save
 
+    assert.ok(
+      query(".user-status-background img").title.startsWith(
+        "off to <img src=''> dentist",
+        "title is properly escaped"
+      )
+    );
```

Comment:
say "no!" to `assert.ok` and `query()` 😛 

```suggestion
    assert
      .dom(".user-status-background img")
      .hasAttribute(
        "title",
        /^off to <img src=''> dentist/,
        "title is properly escaped"
      );
```